docs/installation.rst

```diff
@@ -268,7 +268,10 @@ Prerequisites are installed on **Fedora 20** with::
     $ sudo yum install libtiff-devel libjpeg-devel libzip-devel freetype-devel \
         lcms2-devel libwebp-devel tcl-devel tk-devel
 
+Prerequisites are installed on **Fedora 23** with::
 
+    $ sudo dnf install libtiff-devel libjpeg-devel libzip-devel freetype-devel \
```

Comment:
I think that the redhat-rpm-config is better placed in the earlier section around line 253 where we're installing the python dev libraries. It's not a library dependency as the others in this section are, it's a build configuration dependency. The other items here are the same as for fedora 20, it's just the yum-> dnf change, which when I tested it gave a warning and still functioned.